sal_interfaces/SALGenerics.xml

```diff
@@ -182,16 +119,34 @@
         <Count>1</Count>
     </item>
 </SALCommand>
+<SALCommand>
+    <Subsystem>SALGeneric</Subsystem>
+    <EFDB_Topic>SALGeneric_command_setAccess</EFDB_Topic>
+    <Explanation>http://sal.lsst.org</Explanation>
+    <item>
+        <EFDB_Name>auth</EFDB_Name>
+        <Description>List of authorized users currently allowed to command this CSC.</Description>
```

Comment:
Please choose names that are more descriptive, such as `authUsers` and `noAuthCscs`. The names `auth` and noAuth` sound complementary, but they really aren't because they contain different categories of names.

Reply:
As these were the names specified on the confluence discussion page I guess we will have to 
re-visit it and see if we can agree on changes. I think Robert wants another go-around on the functionality anyway

Reply:
Thank you. I think it is worth some effort to get good names. I thought the page in question was meant as a general design, rather than a specification.